### DISCUSSION TITLE
copilot not working

### DISCUSSION THREAD

**Discussion by shinraxtensei** [community/community#32212] [Copilot Conversations] [answered]
hey copilot is asking me to upgrade even though I have the GitHub student pack 
any solution please?
<img width="1251" alt="Screen Shot 2022-09-07 at 11 03 02 AM" src="https://user-images.githubusercontent.com/48833654/188851336-2378e14e-1706-4378-be29-65be4fd7494b.png">

**Comment by bschaatsbergen** [comment]
Hi Shin, I ran into the same issue; It was solved by re-authenticating for me!

**Reply by shinraxtensei** [reply]
I tried it, but it didn't work 
thanks though

**Reply by bschaatsbergen** [reply]
:( !

**Comment by davecheney** [comment, marked as answer]
@shinraxtensei the message is telling you to install a newer version of the Copilot extension. Please see https://github.com/community/community/discussions/30716.

**Reply by shinraxtensei** [reply]
Ok thanks 🙏
